src/main/java/menu/model/MenuData.java

```diff
@@ -0,0 +1,51 @@
+package menu.model;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import menu.exception.InvalidInputException;
+
+public class MenuData {
+    private static Map<Category, List<String>> menuByCategory = new HashMap<>();
```

Comment:
위의 Map은 상수로 사용해도 될거 같아요!
```suggestion
    private static final Map<Category, List<String>> MENU_BY_CATEGORY = new HashMap<>();
```

Reply:
놓쳤네요 감사합니다!